### PR TITLE
theme: add sonicboom

### PR DESCRIPTION
**Pull request by TheTaylorLee**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description

Created a new theme with a light and dark version

sonicboom

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/sonicboom_dark.omp.json [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                {
+                    "type": "session",
+                    "style": "diamond",
+                    "trailing_diamond": "",
+                    "leading_diamond": "\uE0C5",
+                    "foreground": "#43CCEA",
+                    "background": "#272727",
+                    "properties": {
+                        "postfix": "",
+                        "prefix": " ",
```
Comment (JanDeDobbeleer) [inline themes/sonicboom_dark.omp.json]: this is the default so that can be removed

File: themes/sonicboom_light.omp.json [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                {
+                    "type": "session",
+                    "style": "diamond",
+                    "trailing_diamond": "",
+                    "leading_diamond": "\uE0C5",
+                    "foreground": "#43CCEA",
+                    "background": "#4d4d4d",
+                    "properties": {
+                        "postfix": "",
+                        "prefix": " ",
```
Comment (JanDeDobbeleer) [inline themes/sonicboom_light.omp.json]: this is the default so that can be removed

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
Two remarks

File: themes/sonicboom_dark.omp.json (hunk 1 above)
Comment (TheTaylorLee) [reply themes/sonicboom_dark.omp.json]: Line 97 removed and change committed.

File: themes/sonicboom_light.omp.json (hunk 2 above)
Comment (TheTaylorLee) [reply themes/sonicboom_light.omp.json]: Line 97 removed, and change committed.

**Pull request merged**